OracleTuxedo/core/samples/shm/build.sh

```diff
@@ -2,4 +2,5 @@
 docker build -t oracle/tuxedoshm .
 
 echo "To run the sample, use:"
-echo "docker run -ti -v \${Local_volumes_dir}/TuxedoVolumes/\${VERSION}:/u01/oracle/user_projects oracle/tuxedoshm /bin/bash"
+echo "docker run -d --name tuxedoshm -v \${Local_volumes_dir}/TuxedoVolumes/\${VERSION}:/u01/oracle/user_projects oracle/tuxedoshm /bin/bash"
```

Comment:
Do not ask the user to run `/bin/bash`. Your image should automatically run the appropriate script or process via `CMD` or `ENTRYPOINT`.